docs/EN_US/ContainerizedHPCC/ContainerizedMods/CustomConfig.xml

```diff
@@ -992,4 +991,169 @@ eclagent:
       </sect3>
     </sect2>
   </sect1>
+
+  <sect1 id="LDAPSecurity_forConatiners">
+    <title>LDAP Security for Containers</title>
+
+    <para>Containerized LDAP support for HPCC Systems is accomplished by
+    externalizing the LDAP Active Directory Security Manager administrators
+    account credentials using Kubernetes or Hashicorp Vault secrets. As a
+    prerequisite to use these externalized credentials, you should be familiar
+    with setting up Kubernetes and/or Hashicorp Vault secrets.</para>
+
+    <para>The LDAP Administrator account is the only HPCC user account that
+    must have Active Directory administrator rights. This account should exist
+    in the configured "systemBasedn" branch of the Active Directory, typically
+    set to cn=Users.</para>
+
+    <sect2 id="Configuring_For_KubernetesSecrets">
+      <title>Configuring LDAP in Kubernetes</title>
+
+      <para>To create a secret in Kubernetes for the HPCC Administrator
+      account from a command line intereface to the Kubernetes, execute a
+      command similar to the following example. Note the secret name, is
+      "admincredssecretname" used in this example. The LDAP Administrator
+      "username" and "password" key/values are required; and additional
+      properties are allowed but ignored.</para>
+
+      <programlisting>kubectl create secret generic admincredssecretname --from-literal=username=hpcc_admin \
+  --from-literal=password=t0pS3cr3tP@ssw0rd
+</programlisting>
+
+      <para>This effectively sets the values for the Administrator's username
+      and password in Kubernetes. At this point you can verify the secret you
+      just created by executing the following command on the Kubernetes
+      command line interface. </para>
+
+      <programlisting>kubectl get secret admincredssecretname</programlisting>
+
+      <para>For more information about Kubernetes see the appropriate
+      Kubernetes documentation for your implementation.</para>
+
+      <sect3 id="Deploying_TheKubernetesSecret">
+        <title>Deploying the Kubernetes Secret</title>
+
+        <para>To deploy the Kubernetes secrets override the "secrets:" section
+        in HPCC-Platform/helm/hpcc/values.yaml, or deploy with your own
+        customized chart. For more information about customizing your HPCC
+        Systems containerized deployment see the above sections on
+        customization techniques.</para>
+
+        <para>In your chart, create a unique key name used to reference the
+        secret (this will be the mounted file system name of that secret), and
```

Comment:
The part in ( ) is not needed. This paragraph should reference the example below.